### DISCUSSION TITLE
ClusterMaster to ClusterManager

### DISCUSSION THREAD

**Discussion by cderocco5** [splunk/splunk-operator#1077] [General]
I noticed that in the 2.1 release that the cluster manager crd was created. However, we are early adopters of splunk operator and are currently running 1.1. Will cluster "master" always be supported moving forward or are we forced to move to cluster "manager"

**Comment by vivekr-splunk** [comment]
@cderocco5 ClusterMaster CRD is in V3 and ClusterManager CRD is in V4. we are still supporting ClusterMaster. Also there is a script that converts ClusterMaster and LicenseMaster CR to V4 as ClusterManager/LicenseManager. please use the script https://github.com/splunk/splunk-operator/blob/main/docs/BiasLanguageMigration.md to convert CRs.
